apps/backoffice-v2/src/hooks/useFilter/filter.ts

```diff
@@ -24,13 +24,16 @@ export const filter = <TArray extends AnyArray>({
     return data;
   }
 
-  // Filter
   return data?.filter(item => {
     return terms?.every(term => {
       if (!filter?.[term]?.length) return true;
 
       return filter?.[term]?.some(termValue => {
-        return item?.[term]?.includes(termValue);
+        if (typeof item?.[term]?.includes === 'function') {
+          return item?.[term]?.includes(termValue);
+        }
+
+        return item?.[term] === termValue;
```

Comment:
Refactored this function to support types that don't have `.includes` method, in my specific case I need it for comparing nulls.

Reply:
The structure for a filter is for example  `{ approvalState:  ['NEW', 'APPROVED'] }` which would mean the check is `['NEW', 'APPROVED'].includes('NEW')`, did the check become `null === null` for asignee?

Reply:
If it's unassigned, then yes, it's `null` and should be compared using `===`.